From cc100551b4d92f47abebfa7c7918b2be71263b4a Mon Sep 17 00:00:00 2001
From: Stephen Rothwell <sfr@canb.auug.org.au>
Date: Thu, 15 Sep 2011 17:07:15 +1000
Subject: [PATCH] encrypted-keys: IS_ERR need include/err.h
Git-commit: cc100551b4d92f47abebfa7c7918b2be71263b4a
Patch-mainline: 3.2-rc1

Fixes this build error:

Security/keys/encrypted-keys/masterkey_trusted.c: In function 'request_trusted_key':
security/keys/encrypted-keys/masterkey_trusted.c:35:2: error: implicit declaration of function 'IS_ERR'

Signed-off-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Mimi Zohar <zohar@us.ibm.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 security/keys/encrypted-keys/masterkey_trusted.c |    1 +
 1 file changed, 1 insertion(+)

--- a/security/keys/encrypted-keys/masterkey_trusted.c
+++ b/security/keys/encrypted-keys/masterkey_trusted.c
@@ -16,6 +16,7 @@
 
 #include <linux/uaccess.h>
 #include <linux/module.h>
+#include <linux/err.h>
 #include <keys/trusted-type.h>
 #include <keys/encrypted-type.h>
 #include "encrypted.h"
